config/gateway_paystar.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+return [
+
+    /**
+     *  driver class namespace.
+     */
+    'driver' => Omalizadeh\MultiPayment\Drivers\Paystar\Paystar::class,
+
+    /**
+     *  gateway configurations.
+     * gateway_id is required
+     * type is required => direct | pardakht
+     * use_sign is optional => true | false
+     * secret_key is optional => If use sign is true fill this value, It's your gateway secret key for generate sign
+     */
+    'main' => [
+        'gateway_id' => '',
+        'secret_key' => '',
+        'type' => '',
+        'use_sign' => '',
```

Comment:
if it's bool why you wrote quotes? set a default value

Reply:
Some times developers set secret_key value but they don't need to use sign because it's optional

Reply:
when they don't need to use sign they set it to false. use_sign is bool, isn't it? i asked why the quotes?

Reply:
Done